### PR TITLE
{data}[GCCcore/10.2.0] PostgreSQL 13.2

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-13.2-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    ('Perl', '5.32.0'),
+    ('Python', '3.8.6'),
+]
+
+dependencies = [
+    ('libreadline', '8.0'),
+    ('zlib', '1.2.11'),
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.0.2h'),
```
Comment (Micket) [inline easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-13.2-GCCcore-10.2.0.eb:38]:
```suggestion
    # ('OpenSSL', '1.1.1h'),
```

File: easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-13.2-GCCcore-10.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
```
Comment (Micket) [inline easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-13.2-GCCcore-10.2.0.eb:41]:
```suggestion
osdependencies = [OS_PKG_OPENSSL_DEV]
```

**Review by Micket — CHANGES_REQUESTED** [review]

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/a0f77ea6bc4af96abb2ef13b23e68f95 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0305u15a.bear.cluster - Linux RHEL 8.2, POWER, 8335-GTX (power9le), Python 3.6.8
See https://gist.github.com/1a4eabb321586b85f9b31a25d63d05bc for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @fizwit!

**Pull request merged**